browser/scss/kit/main.scss

```diff
@@ -6,6 +6,7 @@
 	img {
 		max-height: 200px
 	}
+
 }
 
 .kit {
```

Comment:
great thanks!
ᐧ

On Sat, Jun 11, 2016 at 3:10 PM, ARRiera96 <notifications@github.com> wrote:

> In browser/scss/kit/main.scss
> <https://github.com/jdhang/pandoras-kits/pull/66#discussion_r66710306>:
>
> > @@ -6,6 +6,7 @@
> >  	img {
> >  		max-height: 200px
> >  	}
> > +
> >  }
> >
> >  .kit {
>
> just pulled your last changes, it should be gone when I push up.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jdhang/pandoras-kits/pull/66/files/a529c881c470a00fa7287b93bed68ed4de7ddafc#r66710306>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AO_ujXcwkiaYTAzKpIZraX8VcxrA7-3yks5qKwgggaJpZM4IznG_>
> .
>